Source/WebKit/WebProcess/WebPage/AcceleratedSurface.h

```diff
@@ -60,9 +60,15 @@ class AcceleratedSurface {
     virtual void willDestroyGLContext() { }
     virtual void finalize() { }
     virtual void willRenderFrame() { }
+#if ENABLE_WPE_PLATFORM || PLATFORM(GTK)
     virtual void didRenderFrame(WebCore::Region&&) { }
+#else
+    virtual void didRenderFrame() { }
+#endif
 
+#if ENABLE_WPE_PLATFORM || PLATFORM(GTK)
     virtual const WebCore::Damage& addDamage(const WebCore::Damage&) { return WebCore::Damage::invalid(); };
+#endif
```

Comment:
I would move `addDamage()` definition into the first block with preprocessor guards:

```suggestion
#if ENABLE_WPE_PLATFORM || PLATFORM(GTK)
    virtual void didRenderFrame(WebCore::Region&&) { }
    virtual const WebCore::Damage& addDamage(const WebCore::Damage&) { return WebCore::Damage::invalid(); };
#else
    virtual void didRenderFrame() { }
#endif
```